posthog/test/test_utils.py

```diff
@@ -45,6 +59,16 @@ def test_format_absolute_url(self) -> None:
                         msg=f"with URL='{url}' & site_url setting='{site_url}' actual did not equal {expected}",
                     )
 
+    def test_absolute_uri_can_not_escape_out_host(self) -> None:
+        with self.settings(SITE_URL="https://app.posthog.com"):
+            with pytest.raises(PotentialSecurityProblemException):
+                absolute_uri("https://an.attackers.domain.com/bitcoin-miner.exe"),
+
```

Comment:
Thought of one case that isn't covered. What if here you exclude the `https://` from the attacker domain?